scripts/qa_crowbarsetup.sh

```diff
@@ -2303,6 +2315,16 @@ function oncontroller()
     return $?
 }
 
+function get_docker_image()
+{
+    # docker can only be run from SLE12 node
+    local dockernode=$(knife search node "roles:nova-multi-compute-docker" -a name | grep ^name: | cut -d : -f 2 | tail -n 1 | sed 's/\s//g')
+    ssh $dockernode "docker pull cirros && docker save -o cirros-docker.tar cirros"
```

Comment:
That seems correct. In such case, we could drop this get_docker_image function...